core/src/test/java/io/confluent/kafka/schemaregistry/RestApp.java

```diff
@@ -67,6 +66,8 @@ public RestApp(int port,
     prop.put(SchemaRegistryConfig.KAFKASTORE_TOPIC_CONFIG, kafkaTopic);
     prop.put(SchemaRegistryConfig.SCHEMA_COMPATIBILITY_CONFIG, compatibilityType);
     prop.put(SchemaRegistryConfig.LEADER_ELIGIBILITY, leaderEligibility);
+
+    prop.put("metrics.jmx.prefix", "kafka.schema.registry");
```

Comment:
Where is this used?

Reply:
Not needed after rest-utils got fixed, removed.